README.md

```diff
@@ -354,6 +274,126 @@ this would result in a limit of 1256 file descriptors for the system scope.
 Note that we only showed the configuration for the system scope here, equivalent
 configuration options apply to all other scopes as well.
 
+### Default limits
+
+By default the resource manager ships with some reasonable scaling limits and
+makes a reasonable guess at how much system memory you want to dedicate to the
+go-libp2p process. For the default definitions see `DefaultLimits` and
+`ScalingLimitConfig.AutoScale()`.
+
+### Tweaking Defaults
+
+If the defaults seem mostly okay, but you want to adjust one facet you can do
+simply copy the defaults and update the field you want to change. You can
+apply changes to a `BaseLimit`, `BaseLimitIncrease`, and `LimitConfig` with
+`.Apply`.
+
+### How to tune your limits
+
+Once you've set your limits and monitoring (see below) you can now tune your
+limits better.  The `blocked_resources` metric will tell you what was blocked
+and for what scope. If you see a steady stream of these blocked requests it
+means your resource limits are too low for your usage. If you see a rare sudden
+spike, this is okay and it means the resource manager protected you from some
+anamoly.
+
+### How to disable limits
+
+Sometimes disabling all limits is useful when you want to see how much
+resources you use during normal operation. You can then use this information to
+define your initial limits. Disable the limits by using `InfiniteLimits`.
+
+### Debug "resource limit exceeded" errors
+
+These errors occur whenever we've hit a limit. For example we'll get this error
+if we are at our limit for the number of streams we can have, and we try to open
+one more.
```

Comment:
There's an inconsistency between "you" in the other paragraphs and the "we" here. Maybe it's better to use passive voice here: "These errors occur whenever a limit is hit."

Reply:
Used the passive voice for the first sentence, then `you` for the rest of the paragraph.